_includes/configuration/toolbar-persist.md

```diff
@@ -0,0 +1,15 @@
+## toolbar_persist
+
+Use this option to disable the automatic show and hide behaviour of the inline toolbar. Enabling this setting would, by default, always show the inline toolbar and not hide it whenever focus is lost.
+
+**Type:** `Boolean`
```

Comment:
We'll need to add the `{{requires_5_5v}}` note here at some stage, but I don't believe @tylerkelly13 has set that up yet.